cloud/test/recycler_test.cpp

```diff
@@ -309,6 +309,21 @@ static int create_version_kv(TxnKv* txn_kv, int64_t table_id, int64_t partition_
     return 0;
 }
 
+static int create_table_version_kv(TxnKv* txn_kv, int64_t table_id) {
+    auto key = table_version_key({instance_id, db_id, table_id});
+    std::string val(sizeof(int64_t), 0);
+    *reinterpret_cast<int64_t*>(val.data()) = (int64_t) 1;
+    std::unique_ptr<Transaction> txn;
+    if (txn_kv->create_txn(&txn) != TxnErrorCode::TXN_OK) {
+        return -1;
+    }
+    txn->put(key, val);
+    if (txn->commit() != TxnErrorCode::TXN_OK) {
+        return -1;
```

Comment:
Do not swallow the TxnErrorCode here, MetaServiceProxy will rely on this return value to retry.

Reply:
This method is only used for unit test.